packages/auth/amplify_auth_cognito_dart/lib/src/state/event/credential_store_event.dart

```diff
@@ -173,9 +173,6 @@ class CredentialStoreStoreCredentials extends CredentialStoreEvent {
         'Credential store has error. Re-load before continuing.',
       );
     }
-    if (currentState.type != CredentialStoreStateType.success) {
```

Comment:
Was this just straightup not needed? Or is this the result of a change in this PR?

Reply:
It was overly cautious and not technically needed. Queuing is handled appropriately in the state machine already.